zerver/openapi/zulip.yaml

```diff
@@ -103,6 +106,8 @@ paths:
                 $ref: '#/components/schemas/ApiKeyResponse'
   /events:
     get:
+      operationId: get_events
+      tags: ["Real-time events"]
```

Comment:
Hmm.. is there a way to name tags like Python variables? So `real_time_events` instead of `Real-time events`? It makes it easier to add tags when they are formatted consistently and don't expect future contributors to get whitespace and capitalization right. Ideally, we should have a consistent format for tag naming and then these should perhaps be mapped to their proper capitalized versions somewhere before being displayed on the frontend.

Reply:
I actually used the heading of the endpoint in the api docs for the tags, but yes in the future if tags are required for some purpose whitespaces can lead to problems. I will edit it.

Reply:
I wonder if we should use a YAML or OpenAPI variable for these tag names rather than just having them be strings in here?  Would result in better protection against typos.

Reply:
@timabbott Sorry for some reason I discovered this comment a few hours before. I have added tests in this [PR](https://github.com/zulip/zulip/pull/15042) to make sure each endpoint has a valid tag and operationId.